app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt

```diff
@@ -247,6 +248,16 @@ internal class ReleaseMetricController(
             ProgressiveWebApp.installTap.record(NoExtras())
         }
 
+        Component.FEATURE_SEARCH to AdsTelemetry.SERP_ADD_CLICKED -> {
+            BrowserSearch.adClicks[value!!].add()
```

Comment:
I know original code uses `value!!` as well but I think we should change it to `value?.apply`?

Reply:
@rocketsroger I would keep this with the not null assertion because it is an important piece of telemetry we do not want to fail silently.